geo/src/test/java/org/odk/collect/geo/GeoUtilsTest.java

```diff
@@ -52,12 +59,27 @@ public void test_formatLocationResultString() {
     @Test
     public void capitalizesGps() {
         String input = "gps";
-        assertEquals("GPS", GeoUtils.capitalizeGps(input));
+        assertEquals("GPS", capitalizeGps(input));
 
         String locationProvider = "network";
-        assertEquals("network", GeoUtils.capitalizeGps(locationProvider));
+        assertEquals("network", capitalizeGps(locationProvider));
 
         String nullLocationProvider = null;
-        assertNull(GeoUtils.capitalizeGps(nullLocationProvider));
+        assertNull(capitalizeGps(nullLocationProvider));
+    }
+
+    @Test
+    //Cm accuracy #4198
```

Comment:
I think it's best to avoid comments that reference issues like this. Better to do this in the git commit description when adding the code. Let's remove the other `//Cm accuracy #4198` comments as well.